README.md

```diff
@@ -16,33 +16,33 @@
 "NLP.js" is a general natural language utility for nodejs. Currently supporting:
 
 - Guess the language of a phrase
-- Fast levenshtein distance of two strings
-- Search the best substring of a string with less levenshtein distance to a given pattern.
+- Fast _Levenshtein_ distance of two strings
+- Search the best substring of a string with less _Levenshtein_ distance to a given pattern.
 - Get stemmers and tokenizers for several languages.
 - Sentiment Analysis for phrases (with negation support).
 - Named Entity Recognition and management, multilanguage, and accepting similar strings, so the introduced text does not need to be exact.
```

Comment:
This line can be removed, as it's replaced by the next one.

Reply:
Hi @jesus-seijas-sp,

Sorry, do you mean that the "Fast levenshstein ..." line duplicates the "Search the best substring ..." line?

Btw, I realised I introduced a duplicated line, so removed that ... ;)

I have a few fixes for the Table of contents links, then I'll be ready.

Ta,

Nick

Reply:
Hello, the line was "Named Entity Recognition...". It's already fixed.